Cards/MadGraph5_aMCatNLO/SingleT/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+import model loop_sm-no_b_mass
+
+define p = p b b~
+generate p p > t W- [QCD]
+add process p p > t~ W+ [QCD]
+
+output TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8 -nojpeg
```

Comment:
Oh sorry forgot to explain several things and also one mistake

1. TWminus -> TW
2. We ignore bottom that comes from top in the names (e.g. TTto2L2Nu instead of TTto2B2L2Nu) so here you've done right.
3. Decay particles need to be alphabetically ordered (TWto2L2Nu, not TWto2Nu2L)
4. 5FS is default so we ignore this if it's 5FS, only write 4FS if it is 4FS
5. (assuming this is 4FS) I would do TWto2L2Nu-DR1-4FS_blahblah since (in my opinion) DR1 is the more "core physics part" then flavor schemes (so if there are multiple different things to be concatenated together we usually discuss what how to orderly organize it e.g. NJets, MLL, etc.).
6. TuneCP5_13p6TeV are "campaign relevant" so these names get appenedded later on when we replace tune and beamE settings in the back end. so this has to be removed

TWto2L2Nu-DR1_amcatnlo-pythia8 - is what I would do

Reply:
Agreeing on the above. No strong opinion about DR1 and 4FS order. A bit bad for searching. Importance is relative. Maybe alphabetic would be a better argument in this case. Do we already have examples with 2 "restrictions".

Reply:
VV 1Jets 4FS

https://github.com/cms-PdmV/GridpackFiles/pull/43